project/src/main/webapp/discovery.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+  <body style="background c">
```

Comment:
What is background c?

Reply:
It's a style attribute that I should've deleted when I pasted from my portfolio. I'll do that now.